chat-server/serverhtml.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Welcome to CYF Chat</title>
+  </head>
+  <body>
+    <h1>CYF Chat</h1>
+    <h2>Send a message</h2>
+    <form id="messageForm" action="/messages" method="post">
+      <p>
+        Name:
+        <input type="text" name="from" id="from" placeholder="Your Name" />
+        <br />
+        Message:
+        <input type="text" name="text" id="text" placeholder="The message..." />
+        <br />
+      </p>
+      <button type="submit">Send</button>
+    </form>
+
+    <a href="/messages">See all messages</a>
+    <script src="/newserver.js"></script>
```

Comment:
Might be worth trying to remove this and test if it has any difference. Normally script tags are used just for frontend javascript and arent required in this usecase have a look in the console to see if this is imported correct.